### PR TITLE
[TESTING_PURPOSES_ONLY] All languages plus picker fixes

### PR DESCRIPTION
**Pull request by ice-hector**

### CODE REVIEW

File: src/screens/HomeFlow/Home/components/Tasks/components/ProgressItem/index.tsx [hunk 1]
```diff
@@ -57,25 +57,28 @@ const styles = StyleSheet.create({
...
   },
   description: {
     marginTop: rem(4),
-    ...font(12, 14, 'medium', 'toreaBay'),
+    ...font(12, 16, 'medium', 'toreaBay'),
   },
   progress: {
     position: 'absolute',
   },
+  titleDescriptionWrapper: {
```
Comment (ice-atlas) [inline src/screens/HomeFlow/Home/components/Tasks/components/ProgressItem/index.tsx:81]: use `commonStyles.flexOne`

File: src/store/modules/Account/sagas/getAccount.ts [hunk 2]
```diff
@@ -11,11 +11,24 @@ export function* getAccountSaga() {
...
+    const existingUser: ReturnType<typeof userSelector> = yield select(
+      userSelector,
+    );
     const user: SagaReturnType<typeof Api.user.getUserById> = yield call(
       Api.user.getUserById,
       userId,
     );
     yield put(AccountActions.GET_ACCOUNT.SUCCESS.create(user));
+
+    const currentUserLanguage = existingUser?.language;
```
Comment (ice-orion) [inline src/store/modules/Account/sagas/getAccount.ts:23]: let's revert these changes, since it's handled in https://github.com/ice-blockchain/mobile-app/pull/93 ?
